components/org.wso2.carbon.identity.api.user.governance/src/main/java/org/wso2/carbon/identity/user/endpoint/impl/MeApiServiceImpl.java

```diff
@@ -76,7 +76,10 @@ public Response getMe() {
             userAttributes = Utils.getUserInformationService().getRetainedUserInformation(username, userStoreDomain,
                     tenantId);
         } catch (UserExportException e) {
-            return Response.serverError().entity(e.getMessage()).build();
+            ErrorDTO errorDTO = new ErrorDTO();
+            errorDTO.setRef(Utils.getCorrelation());
+            errorDTO.setMessage(e.getMessage());
+            return Response.serverError().entity(errorDTO).build();
```

Comment:
Are you changing the response structure here?

Reply:
Yes. The response body will have a new field **traceId**